controllers/common.go

```
@@ -211,14 +214,110 @@ func (r *MultiClusterHubReconciler) ensureOperatorGroup(m *operatorv1.MultiClust
+
+	if !errors.IsNotFound(err) {
+		// Unexpected error
+		r.Log.Info(fmt.Sprintf("error getting ServiceAccount in ns: %s. Error: %s", sa.GetNamespace(), err.Error()))
+		return ctrl.Result{Requeue: true}, nil
+	}
+
+	// ServiceAccount doesn't exist, create it
+	r.Log.Info(fmt.Sprintf("Ensuring ServiceAccount exists in ns: %s", sa.GetNamespace()))
+
+	force := true
+	err = r.Client.Patch(ctx, sa, client.Apply, &client.PatchOptions{Force: &force, FieldManager: "multiclusterhub-operator"})
```

golangci-lint (2.12.2)

[error] SA1019: client.Apply is deprecated: Use client.Client.Apply() and client.Client.SubResource("subrsource").Apply() instead. 

(staticcheck)